docs/sources/auth/planning/index.md

```diff
@@ -120,7 +120,11 @@ Service account access tokens inherit permissions from the service account.
 
 ### 🚧 Personal access tokens
 
-### 🚧 API keys
+### API keys
+
+> **Note:** If you use Grafana v8.5 or newer, you should use service accounts instead of API keys. API keys will be deprecated in the near future. For more information, refer to [Grafana service accounts]({{< relref "../service-accounts/" >}}).
+
+You can use Grafana API keys to interact with data sources via HTTP APIs. API keys can have a well-defined and limited scope to resources with the help of [Roles].
```

Comment:
API keys aren't limited to data sources - they can be used to interact with any Grafana resource.